src/stream/stream_backend.rs

```diff
@@ -1,10 +1,9 @@
-use super::types::*;
 use super::video_stream_rtsp::VideoStreamRtsp;
 use super::video_stream_udp::VideoStreamUdp;
+use super::{types::*, video_stream_redirect::VideoStreamRedirect};
```

Comment:
This should be done in a169165c30f1c1d99b988fc23ec85a24dc3fcda5

Reply:
My point that this line is just organized a line added in a169165.

Reply:
It seems like, but actually, it is adding `supper::types::*`, which should be only the `supper::types::StreamType`.